driver/driver.hpp

```diff
@@ -150,7 +150,8 @@ inline void PadBufferSize(size_t& sz, int datatype_sz)
     printf("Supported Base Arguments: conv[fp16|int8|bfp16|fp8|bfp8], CBAInfer[fp16], "
            "pool[fp16], lrn[fp16], "
            "activ[fp16], softmax[fp16], bnorm[fp16], rnn[fp16], gemm[fp16], ctc, dropout[fp16], "
-           "tensorop[fp16], reduce[fp16|fp64], layernorm[bfp16|fp16], sum[bfp16|fp16], "
+           "tensorop[fp16], reduce[fp16|fp64], layernorm[bfp16|fp16], "
+           "marginrankingloss[bfp16,fp16], sum[bfp16|fp16], "
```

Comment:
You'll need to modify this anyway after merging or rebasing from the newest develop, but you should add new things at the last position, after `cat`.
It's either last position or alphabetical order, and it's clearly not the latter case.

Reply:
This goes same to `docs/reference/index.rst`.

Reply:
Understood